packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -272,6 +272,35 @@ const fn = new lambda.Function(this, 'MyFunction', {
 See [the AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/concurrent-executions.html)
 managing concurrency.
 
+### AutoScaling
+
+You can use Application AutoScaling to automatically configure the provisioned concurrency for your functions. AWS Lambda supports Target Tracking and Scheduled Scaling policies. Use the `addAutoScaling()` method to configure autoscaling on a function alias:
```

Comment:
You can use Application AutoScaling to automatically configure the required provision concurrency for your functions. AutoScaling can be set to track utilization or based on schedule. To configure AutoScaling on a function alias: